source/MenuPanel.cpp

```diff
@@ -177,6 +177,13 @@ void MenuPanel::OnCallback(int)
 
 
 
+bool MenuPanel::AllowFastForward() const
+{
+	return true;
```

Comment:
Why would anyone want to toggle fast-forward while being in the MenuPanel?

Reply:
You start the game in the MenuPanel. If we disable fast-forward in the menu panel, players new to this will get confused and think fast-forward is broken.

Reply:
I found a better solution to that particular problem. Now, fast-forward is only available in space, and you'll get a warning dialog if you try to enable it elsewhere. The dialog only shows up once per program execution.